File: 1293-three-consecutive-odds/1293-three-consecutive-odds.kt
class Solution {
    fun threeConsecutiveOdds(arr: IntArray): Boolean {
        var count = 0
        for (a in arr) {
            if (a%2!=0) {
                count++
            } else {
                count = 0
            }
            if (count == 3) {
                return true
            }
        }
        return false
    }
 }

File: 1293-three-consecutive-odds/1293-three-consecutive-odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        count = 0
        for value in arr:
            if value%2 != 0:
                count+=1
            else:
                 count = 0
            if count == 3:
                return True
        return False